### ISSUE TITLE
Give it the ol' Playtonic Spit-n-Polish

### ISSUE THREAD

**Issue by EnMod** [EnMod/noel-quiles-www#1]
Revisit the project, add any missing features/enhancements/bugfixes that would help it (actual accessibility, better navigation, a background, etc.), then submit all at once, much like Yooka-Laylee's first update.

**Comment by EnMod** [comment]
Still working on the accessibility part, that's next

**Comment by EnMod** [comment]
Accessibility/any other changes will be a focus of the new redesign! Check out #12 for more on that as it happens.
